packages/kbn-storybook/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "@kbn/storybook",
+  "author": "Operations",
```

Comment:
Sync'd with Stacey on this one - we're going to take ownership for now until we can identify another team more suited to improve storybook usage.